.github/workflows/job_PreCommit_PythonAutoformatter.yml

```diff
@@ -0,0 +1,27 @@
+name: Precommit Python Formatter
+run-name: Python Formatter
+on:
+  pull_request:
+    branches: [ "master", "release-*" ]
+    paths: [ "sdks/python/apache_beam/.*"]
+  issue_comment:
+    types: [created, edited]
+
+
+jobs:
+  PythonFormatter:
+    if: github.event.issue.pull_request && contains(github.event.comment.body, 'Run PythonLint PreCommit')
```

Comment:
Should it be the comment body is exact 'Run PythonLint PreCommit' (after trimming whitespaces)?